app/views/nsm/disbursements/_adjusted.html.erb

```diff
@@ -1,5 +1,7 @@
-<% if records.size.zero? %>
-  <p><%= t('.no_data') %></p>
+<% if claim.data['disbursements'].blank? %>
```

Comment:
```ruby
if claim.data['disbursements'].blank? || records.empty?
```

Reply:
In this context `records` are adjusted disbursements, and we _don't_ want to show `.no_disbursements` if `claim.data['disbursements']` is not blank but `records` is empty, as that implies there are disbursements but none of them are adjusted. So in that scenario we want to show `.no_adjustments`

Reply:
I misread completely and thought they both were `.no_disbursements` 🤣